boostregex/AnsiDocumentIterator.h

```diff
@@ -1,13 +1,35 @@
-﻿#ifndef ANSIDOCUMENTITERATOR_H_12481491281240
+﻿// This file is part of Notepad++ project
+// Copyright (C) 2021 Notepad++ authors.

```

Comment:
Next time you can use "Anonymous" or "Nobody" instead of "Notepad++ authors" if you don't want to leave your name.
BTW, hiding your name could be a solution. My way is:
"Always code as if the guy who ends up maintaining your code will be a violent psychopath who knows where you live." 
;)

Reply:
> Always code as if the guy who ends up maintaining your code will be a violent psychopath who knows where you live.

Yep. This is exactly what I have in mind.
I don't want to be sued for accidentally breaking some contract conditions😄

But honestly, I wasn't the author of this file. So I just added a "generic" comment header.